common/commands/config.go

```diff
@@ -626,6 +632,47 @@ func (cc *ConfigCommand) encryptPassword() error {
 	return err
 }
 
+// Assert all services URLs are safe
+func (cc *ConfigCommand) assertUrlsSafe() error {
+	for _, url := range []string{cc.details.Url, cc.details.AccessUrl, cc.details.ArtifactoryUrl,
+		cc.details.DistributionUrl, cc.details.MissionControlUrl, cc.details.PipelinesUrl, cc.details.XrayUrl} {
+		if isUrlSafe(url) {
+			continue
+		}
+		if cc.interactive {
+			if cc.disablePrompts || !coreutils.AskYesNo("We noticed that your JFrog URL uses an insecure HTTP connection. Are you sure you want to continue?", false) {
```

Comment:
We noticed that your JFrog URL uses an insecure HTTP connection. Are you sure you want to continue?
-->
Your JFrog URL uses an insecure HTTP connection, instead of HTTPS. Are you sure you want to continue?